applications/consdb/templates/deployment.yaml

```diff
@@ -45,24 +45,28 @@ spec:
               value: "{{ .Values.db.user }}"
             - name: "DB_NAME"
               value: "{{ .Values.db.database }}"
-            - name: "LFA_USER"
-              value: "{{ .Values.lfa.user }}"
-            - name: "LFA_PASS"
+            - name: "AWS_ACCESS_KEY_ID"
+              value: "{{ .Values.lfa.access_key }}"
```

Comment:
i had https://rubinobs.atlassian.net/browse/DM-43284  for one of the new people to fix this - so I think we can close that also ?